src/cart/crt_iv.c

```diff
@@ -1627,20 +1749,26 @@ crt_iv_fetch(crt_iv_namespace_t ivns, uint32_t class_id,
 		/* We got error, call the callback and exit */
 		iv_ops->ivo_on_refresh(ivns_internal, iv_key, 0,
 				NULL, false, rc, user_priv);
+
 		fetch_comp_cb(ivns_internal, class_id, iv_key, NULL,
 			      NULL, rc, cb_arg);
+
 		iv_ops->ivo_on_put(ivns_internal, iv_value, user_priv);
 		D_FREE_PTR(iv_value);
 
 		/* ADDREF done above in lookup */
 		IVNS_DECREF(ivns_internal);
+		DBG_EXIT();
 		return rc;
 	}
-
+	/* The request is not located on current server.	*/
+	/* Create an rpc request to external server.		*/
 	/* Return read-only copy and request 'write' version of iv_value */
+	/* Free up previos iv_value structure.			*/
```

Comment:
multi-line coment should have same style as copyright message (see start of file)

Reply:
Fixed